x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/inference_config.tsx

```diff
@@ -23,10 +22,10 @@ import { ZeroShotClassificationInferenceConfiguration } from './zero_shot_infere
 export const InferenceConfiguration: React.FC = () => {
   const {
     addInferencePipelineModal: { configuration },
-    selectedMLModel,
+    selectedModel,
   } = useValues(MLInferenceLogic);
-  if (!selectedMLModel || configuration.existingPipeline) return null;
-  const modelType = getMLType(getMlModelTypesForModelConfig(selectedMLModel));
+  if (!selectedModel || configuration.existingPipeline) return null;
+  const modelType = getMLType([selectedModel.type]);
```

Comment:
I think this could be simplified by using `selectedModel.type` directly. I'll check if there are any side effects and modify the code in a separate PR.

Reply:
Good call - it does change the type from `lang_ident` to `classification`. I think that's OK - the built-in language identification model's type _is_ classification, and I don't see any logic that depends on the value of `modelType`. But for the sake of less moving parts I [restored](https://github.com/elastic/kibana/pull/176257/commits/27bc3b0d4b2c9ee315108ba1c9e9ad9fb0ca2bb6) the original value parsing in this line and in another.